.github/workflows/aws-sqs-moduleTest.yml

```diff
@@ -68,56 +33,44 @@ jobs:
         shell: bash
 
     steps:
-#     # Checkout the repository to the GitHub Actions runner
-#     - name: Checkout
-#       uses: actions/checkout@v2
+      # Checkout the repository to the GitHub Actions runner
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Install the latest version of Terraform CLI and configure the Terraform CLI configuration file with a Terraform Cloud user API token
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+
+      # Export aws secret and access key as vars
+      - name: Export secre_key, access_key and region
+        with:
+          export TF_VAR_region: ${{ secrets.MAVEN_AWS_REGION }}
+          export TF_VAR_access_key: ${{ secrets.MAVEN_AWS_ACCESS_KEY }}
+          export TF_VAR_secret_key: ${{ secrets.MAVEN_AWS_SECRET_KEY }}
+
+      ## Using terraform implementation in workflow above. Can be changed to AWS configuration below.
 
-#     # Install the latest version of Terraform CLI and configure the Terraform CLI configuration file with a Terraform Cloud user API token
-#     - name: Setup Terraform
-#       uses: hashicorp/setup-terraform@v1
-      
-#     # Export aws secret and access key as vars
-#     - name: Export secre_key, access_key and region
-#       with:
-#         export TF_VAR_region: ${{ secrets.MAVEN_AWS_REGION }}
-#         export TF_VAR_access_key: ${{ secrets.MAVEN_AWS_ACCESS_KEY }}
-#         export TF_VAR_secret_key: ${{ secrets.MAVEN_AWS_SECRET_KEY }}
-    
-#     ## Using terraform implementation in workflow above. Can be changed to AWS configuration below.
-    
-# #     - name: Configure AWS Credentials
-# #       uses: aws-actions/configure-aws-credentials@v1
-# #       with:
-# #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-# #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-# #         aws-region: us-west-1
+      #     - name: Configure AWS Credentials
+      #       uses: aws-actions/configure-aws-credentials@v1
+      #       with:
+      #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      #         aws-region: us-west-1
```

Comment:
if we are not using this can you clean it up?

Reply:
Cleaned up.